sdk/go/sdk.go

```diff
@@ -93,6 +96,23 @@ type Channel struct {
 	Metadata string  `json:"metadata,omitempty"`
 }
 
+// Message represents mainflux message.
+type Message struct {
```

Comment:
Why are we duplicating the code instead of having shared libs?

Reply:
@dusanb94 what's your opinion here?